lib/Mojo/Message/Request.pm

```diff
@@ -301,6 +302,13 @@ Proxy URL for request.
 
 Request has been performed through a reverse proxy.
 
+=head2 trusted_proxies
+
+  my $proxies = $req->trusted_proxies;
+  $req        = $req->trusted_proxies(['10.0/8', '127.0.0.1', '172.16.0/12', '192.168.0/16']);
```

Comment:
Could there be a IPv6 address in this example just to make it more obvious v6 supported?

Reply:
done